incubator/java-openliberty/image/project/invoke-appsody.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [ -e /project/user-app/.appsody-nolocal ]
```

Comment:
For now let's call it `.appsody-binary`.   If needed we could broaden this and rename to something like this..For now it might be more obvious to tie it to binary.

Reply:
done